core/src/main/scala/kafka/coordinator/transaction/TransactionMarkerChannelManager.scala

```diff
@@ -156,6 +155,9 @@ class TransactionMarkerChannelManager(config: KafkaConfig,
   }
 
   // visible for testing
+  private[transaction] def queueForUnknownBroker = markersQueueForUnknownBroker
```

Comment:
Hmm, I felt the variable names the queue `markersQueueForUnknownBroker` and `markersQueuePerBroker` are okay, and the reason of these two functions `queueForUnknownBroker` and `queueForBroker()` is exactly to separate the actual private variables from the test-only functionalities.